### PR TITLE
Calendar JS Functionality 

### LINKED ISSUE #62: Calendar Add Sign Easter Egg
Need to get rid of the add-sign that pops up in calendar in the mobile view

### LINKED ISSUE #68: Calendar JS rework
Overall Rework
- adding events such as meetings

### PR DESCRIPTION
**Pull request by KaustubhPaliwal**
## Calendar - Pull Request

**Description**
Completed Calendar with the following features:
- Ability to sync in Tasks from the TaskList
- Can create new events 
- Highlights dates with Tasks and Underlines them if they have an Event
- Can edit, delete, or create a new event 
- Automatically opens up to the current date 

**Type of change**
- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

**Checklist**
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

**Related Issues**
Fixes #62, Fixes #68

### CODE REVIEW

File: src/scripts/main/public/html/addEvent.html
Comment (kevku) [inline src/scripts/main/public/html/addEvent.html:1]: New Functionality to add an event to calendar

File: src/scripts/main/public/html/team_calendar.html
Comment (kevku) [inline src/scripts/main/public/html/team_calendar.html:1]: Modifies the calendar to add events and gives room to dynamically add the tasks

**Review by erhuang623 — APPROVED** [review]
Good work! Integrating other functionalities into the calendar is crucial to the calendar.

**Review by kevku — APPROVED** [review]
Added new functionalities of events and cross feature of tasklist

**Pull request merged**